script/export_for_foi/export_sites.sql

```diff
@@ -0,0 +1,12 @@
+COPY (
+  SELECT
+    abbr AS "Abbreviation",
+    launch_date AS "Launch Date",
+    homepage AS "New Homepage",
+    query_params AS "Significant querystring parameters (colon separated)",
+    CASE global_type WHEN 'unresolved' THEN 410 WHEN 'archive' THEN 410 WHEN 'redirect' THEN 301 END AS global_http_status,
+    global_new_url AS "New URL for global 301",
+    CASE global_redirect_append_path WHEN 'f' THEN 'false' WHEN 't' THEN 'true' END AS "Append requested path to new path? (for global 301)"
```

Comment:
I think `"Append requested path to new URL? (for global 301)"` would be more accurate here.